src/Microsoft.Health.Dicom.Core/Features/Query/QueryResponseBuilder.cs

```diff
@@ -32,7 +32,19 @@ public class QueryResponseBuilder
         DicomTag.StudyID,
     };
 
-    private static readonly HashSet<DicomTag> AllStudyTags = new HashSet<DicomTag>(DefaultStudyTags)
+    internal static readonly HashSet<DicomTag> V2DefaultStudyTags = new HashSet<DicomTag>()
```

Comment:
Now that we have the computed attributes available, like ModalitiesInStudy, should we include them here and for Series? They are required to be returned, and that could help simplify the logic in the QueryService.

Reply:
I want to keep the default to minimum and fastest and they can always add what they want in includeField